### DISCUSSION TITLE
LoRA and Fooocus

### DISCUSSION THREAD

**Discussion by Megasherru** [lllyasviel/Fooocus#3078] [Q&A] [answered]
Two things, 
1. Im noticing that LoRA dont actually activate when you put them in the prompt, you can test a style and place it in the prompt and it wont activate, it only activates when you use it the normal way though the 5 LoRA sections. <lora:loraname:1> does not activate. 

2. How do you increase LoRA count, I want more than just 5 Lora sections as well. 5 is way too little 
I tried to change the Config file but it wont update in platform

![image](https://github.com/lllyasviel/Fooocus/assets/154871283/29c7f37d-92e6-4696-80a7-5b2de9479677)

**Comment by Rogala** [comment]
Every line in config.txt except the last line is terminated with a comma.

**Comment by mashb1t** [comment]
@Megasherru it should work, there are also tests for this (and only this) feature: https://github.com/lllyasviel/Fooocus/blob/main/tests/test_utils.py

**Comment by Megasherru** [comment]
Update: I solved issue 2, thank you guys. 
I placed the command at the top instead of the bottom and did it with a comma, it worked. 

![image](https://github.com/lllyasviel/Fooocus/assets/154871283/cd7a2f75-d0b8-4984-b053-1e471832eaa1)
![image](https://github.com/lllyasviel/Fooocus/assets/154871283/9738b7ca-866b-40e6-b9bb-bebb465c5f08)



Ill just test Issue 1,
seems the console is asking me to update Gradio, not sure how to do that, but ill find out quick

![image](https://github.com/lllyasviel/Fooocus/assets/154871283/48a9d252-395b-4ab4-a227-bab6479ab026)

**Reply by mashb1t** [reply, marked as answer]
https://github.com/lllyasviel/Fooocus/issues/2911#issuecomment-2108362279

**Reply by Megasherru** [reply]
Thank you for the answer here as well. 
I have now solved all issues.

I think I did something wrong the first time, the Lora in the prompt does work as intended. :)
